app/views/rebate_forms_mailer/_default.haml

```diff
@@ -0,0 +1,27 @@
+%p
+  Hi
+  = @rebate_form.full_name
+
+%p
+  Your application for a rates rebate for the 1 July 2018 – 30 June 2019 rates
+  year has been sent to the #{@council.name} City Council.
+
+%p
+  %b What you need to do now
+
+%p
+  %ul
+    %li Go to the a council service centre
+    %li Tell the Service Centre staff you're there to sign your rates rebate application.
+    %li Evidence of income helps to ensure you receive the correct rebate promptly.
+    %li If you are self-employed, you must supply income evidence with your application.
+
+%p
+  Please note: you can only apply for a rebate for the 2018/2019 rates year between
```

Comment:
@Br3nda same for here, should we use variables for the year

Reply:
I initially made helpers for this, but then found the rebate_form_helpers aren't available in the mailer, so i went back to hard coded because i'm running out of time to get this live